include/Core/DB/TableManager.php

```diff
@@ -167,6 +187,11 @@ public function fetch_log_items_by_id( $ids = array() ) {
 
 		$query = "SELECT * FROM {$table_name}";
 
+		// When `$this->date_column_format` exists, should replace the `$query` var.
+		if ( ! empty( $this->date_column_format ) ) {
```

Comment:
Let's not store date format as a class variable. Instead the date format as an optional variable.

Also alternatively we can do the conversation in PHP also.

Reply:
@sudar,

This comment has been fixed. I've verified that my updated changes didn't break the flow. The CSV I downloaded during my testing is here https://bulkwp.slack.com/archives/G111H3AAV/p1540444220000100